.vscode/settings.json

```diff
@@ -2,7 +2,6 @@
   "files.insertFinalNewline": true,
   "files.trimFinalNewlines": true,
   "files.trimTrailingWhitespace": true,
-  "editor.formatOnSave": true,
```

Comment:
Why was this removed?

Reply:
@andrewthauer For me the auto-format is not adhering to the style conventions, so it was formatting everything incorrectly. I got sick of accidentally saving without that manually disabled and losing all my changes in a sea of formatting adjustments. I can fix it soon, but for the moment it's not worth the hassle.